src/main/java/frc/robot/Constants.java

```diff
@@ -15,5 +15,16 @@
 public final class Constants {
   public static class OperatorConstants {
     public static final int kDriverControllerPort = 0;
+  
   }
+  /*
+  All changes in the constants file are for testing and will be integrated into the larger constants file
+  further down the line by the programming team :)
+  */
+  
+  public static final int INTAKEMOTOR = 9; 
```

Comment:
Use CAPS_UNDERSCORE naming conventions for constants

Reply:
After you merge group this with the other constants for motors